### DISCUSSION TITLE
Colour for compound

### DISCUSSION THREAD

**Discussion by RapidDestroyer1** [theOehrly/Fast-F1#232] [Q&A] [answered]
Is there a way for the colour of the point on a plot of lap time vs lap number to equal the compound the driver is one at that lap?
Thanks.

**Comment by theOehrly** [comment, marked as answer]
The compound names are given in `Session.laps`/`Laps`. The colours are not available through FastF1, but it should be fairly straightforward to figure them out. Then you just need to map them based on the tyre compound name. 
Then, for how you best color individual segments of a plot, you need to take a look at the matplotlib documentation. But it is sure possible to do what you want to do.
